src/cev/_embedding_comparison_widget.py

```diff
@@ -20,15 +20,15 @@
 from cev._compare_zoom_toggle import create_zoom_toggle
 from cev._embedding import Embedding
 from cev._widget_utils import add_ilocs_trait, parse_label
-from cev.components import MarkerSelectionIndicator
+from cev.components import MarkerSelectionIndicator, WidthOptimizer
 
 
 def _create_titles(
     titles: tuple[str, str]
 ) -> tuple[ipywidgets.Widget, ipywidgets.Widget]:
     left_title, right_title = titles
     spacer = ipywidgets.HTML(
-        value='<div style="height: 1px; background: #efefef;" />',
+        value='<div style="height: 1px; background: var(--jp-border-color0);" />',
```

Comment:
Might have an issue with this outside of JupyterLab, but I think it's fine for now.

Reply:
Yeah, for now I consider this widget to be Jupyter only